### PR TITLE
Skeleton version

### PR DESCRIPTION
**Pull request by Theodore354**
This project was done today.It follows the github workflows and the necessary commits.
It was done in the feature branch before been merged. Feel free to bring your feedback.

### CODE REVIEW

**Comment by Theodore354** [comment]
> Microverse Project

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="logo-sec">
+      <a href="#" class="wel">Welcome</a>
+      <div class="menu">
```
Comment (D4nielJ) [inline index.html]: - [ ] Let's change this `div` tag for a `nav` tag. This is good for semantics and accessibility purposes. It makes your code more legible for both humans and machines. [You can read more here.](https://www.freecodecamp.org/news/semantic-html5-elements/)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        <div class="icon3">
+          <img src="medium.svg" alt="medium" />
+        </div>
+        <div class="icon4">
+          <img src="github.svg" alt="github" />
+        </div>
+        <div class="icon5">
+          <img src="angellist.svg" alt="angelist" />
+        </div>
+      </div>
```
Comment (D4nielJ) [inline index.html]: - [ ] Let's change all of the `divs` in this block for an Unordered List and Lists Items. For this, we will use an `ul` tag to wrap the items of the list, and `li` tags to wrap the inner items.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    </section>
```
Comment (D4nielJ) [inline index.html:122]: - [ ] Let's add the forward button before we close this section. The button I'm referring to is the one below the social media icons:

image

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @Theodore354, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: :fire: :fire: 

- You follow GitHub correctly and your repo looks professional
- Your UI is really good so far! Keep it up

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — APPROVED** [review]
# STATUS: APPROVED 🟢 
Hi @Theodore354, 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

## Project highlights:

- [x] Your UI looks identical to the original Figma design :clap: 
- [x] Your commit messages are descriptive and you made the correct use of GitHub Flow.
- [x] All linters checks are passing :heavy_check_mark: 
- [x] Your PR title is descriptive and you have a descriptive summary of all the changes in this branch :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**